aten/src/ATen/native/mps/OperationUtils.h

```diff
@@ -51,8 +56,9 @@ std::string getTensorsStringKey(const TensorList& tensors, bool use_scalar_value
 double getMPSScalarValue(const Tensor& t);
 std::string getArrayRefString(const IntArrayRef s);
 std::string getStridedKey(const Tensor& self, const IntArrayRef sz,
-                          const IntArrayRef strides, int64_t offset);
+                          const IntArrayRef strides, int64_t offset, GatherScatterViewOpType viewOpType);
 id<MTLBuffer> gatherViewTensor(const at::Tensor& src, id<MTLBuffer> s);
+id<MTLBuffer> scatterViewTensor(at::Tensor& output, const at::Tensor src, id<MTLBuffer> updatesTensorBuffer);
```

Comment:
Is there a benefit of passing `src` by value rather than by reference?
```suggestion
id<MTLBuffer> scatterViewTensor(at::Tensor& output, const at::Tensor& src, id<MTLBuffer> updatesTensorBuffer);
```

Reply:
Agreed, fixed in https://github.com/pytorch/pytorch/pull/80491